src/components/ProjectConfigurationModal/ProjectConfigurationModal.js

```diff
@@ -158,28 +162,26 @@ const ProjectIconWrapper = styled.div`
 const SelectableImageWrapper = styled.div`
   display: inline-block;
   margin: 0px 10px 10px 0px;
+`;
 
-  @media ${BREAKPOINTS.sm} {
-    /* Reduce amount of displayed icons on smaller screens.
-       Todo: Check if this is really a good idea, as the currently selected icon could be hidden.
-    */
-    &:nth-of-type(n + 9) {
-      display: none;
-    }
-  }
+const Actions = styled.div`
+  text-align: center;
+  padding-bottom: 16px;
 `;
-const Description = styled.div`
-  font-size: 24px;
-  color: ${COLORS.gray[600]};
+
+const DisabledText = styled.div`
+  padding-top: 16px;
+  color: ${COLORS.gray[500]};
 `;
 
 const mapStateToProps = (state, ownProps) => {
   const project = getSelectedProject(state);
   const projectId = project && project.id;
+
   return {
     project,
-    isVisible: state.modal === 'project', // todo: refactor this so we're having each modal-string in a constant 'project' | 'app' | 'new-project-wizard'
```

Comment:
I mentioned earlier - I removed this `TODO` because I think we can rely on Flow for this :)